meinberlin/apps/offlineevents/forms.py

```diff
@@ -1,7 +1,7 @@
 from django import forms
 from django.utils.translation import ugettext_lazy as _
 
-from meinberlin.apps.datetimefield.widgets import DateTimeInput
+from adhocracy4.forms.widgets import DateTimeInput
```

Comment:
why are we using the widget here instead of the field? accident or decision?

Reply:
I was not sure and kept it as it was.